### DISCUSSION TITLE
Filter the code block floating menu on keydown

### DISCUSSION THREAD

**Discussion by bcbogdan** [outline/outline#6496] [Ideas]
Hello again,

This one's a bit more straightforward. 

- You open the **languages floating menu** from a `code block` node
- You press a letter key
- The list changes the active option to the first one that starts with that letter (scrolls if necessary)
- If the key is pressed again, and multiple options start with that letter, the focus will move to the next one.

Again, if you're ok with the proposal, I can work on a PR for it.

**Comment by tommoor** [comment]
The menu is built using the component system from reakit, it's the same menu that's used everywhere else in the product – so in terms of accessibility improvements this might be best waiting for a move to Radix (https://github.com/outline/outline/issues/6456) to address this at system level
